lms/djangoapps/lms_xblock/runtime.py

```diff
@@ -284,4 +284,8 @@ def applicable_aside_types(self, block):
         if block.scope_ids.block_type in config.disabled_blocks.split():
             return []
 
-        return super(LmsModuleSystem, self).applicable_aside_types()
+        return [
+            aside_type
+            for aside_type in super(LmsModuleSystem, self).applicable_aside_types(block)
+            if aside_type != 'acid_aside'
```

Comment:
We should move that line that installs `acid-block` into a separate requirements.txt file that is only used for tests. Unfortunately, that's likely to require a larger set of test infrastructure changes, so I'll postpone it for now. Can you just add a comment stating that this should be removed once AcidBlock is only installed during tests?